### PR TITLE
feat: ansible 2.15.1

### LINKED ISSUE #280: Update Ansible version from ansible-2.9 to ansible-core-2.15
See https://github.com/TOSIT-IO/tdp-collection/issues/774

### PR DESCRIPTION
**Pull request by gonzaloetjo**
#### Which issue(s) this PR fixes

Fixes #280 and #263

#### Additional comments

- Use ansible-core 2.15.1
- Install collection dependencies on `ansible_dependencies` through the setup script and `requirements.yml`

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-ansible==2.9.27
+ansible-core==2.15.1
 jmespath==0.10.0
```
Comment (kpgtek) [inline requirements.txt:2]: ```jmespath``` is a requirement for Prometheus.
If we use Python 3.9, I don't know if this version 0.10.0 would be compatible.
On my system with Python 3.9, the version 1.0.1 is installed.

**Comment by PACordonnier** [comment]
I dont think we should accept the PR as is. The instructions in the README are not in sync with the current configured submodules (even after https://github.com/TOSIT-IO/tdp-getting-started/pull/286). For example `tdp plan dag` is not implemented in stable.

I understand the need of the PR (add instructions for the current stable tdp-lib and ansible 2.15) but it should include the bump in the submodules as well to work.

I suggest we this PR to a dev branch, this way people who `git clone` master will have a working version.

File: requirements.txt (hunk 1 above)
Comment (gonzaloetjo) [reply requirements.txt]: I added the 1.0.1 version

**Comment by gonzaloetjo** [comment]
@rpignolet @PaulFarault I added the `dev` branch where this should be merged. Also added a note on the top of `README.md` warning that it's a dev branch and some components may not work correctly (currently it's server and ui, but may be something different at an other point).

**Review by PACordonnier — APPROVED** [review]

**Pull request merged**